src/fides/api/ops/api/v1/endpoints/connection_endpoints.py

```diff
@@ -112,6 +112,7 @@ def get_connections(
     disabled: Optional[bool] = None,
     test_status: Optional[TestStatus] = None,
     system_type: Optional[SystemType] = None,
+    orphaned_from_system: Optional[bool] = False,
     connection_type: Optional[List[str]] = Query(default=None),  # type: ignore
 ) -> AbstractPage[ConnectionConfig]:
```

Comment:
Would it be useful to add `system_id` to the response here, or the response to GET /connection, when you're accessing connections directly and not through a system?

Reply:
that's a good point about the id versus fides_key, I forgot about that.